python/lsst/ci/builder/commands.py

```diff
@@ -71,8 +71,7 @@ class IngestRaws(BaseCommand):
     """
 
     def run(self, currentState: BuildState):
-        ingestRaws(self.runner.RunDir, (self.rawLocation,), self.FITS_RE, None,
-                   processes=int(self.arguments.num_cores))
+        ingestRaws(self.runner.RunDir, (self.rawLocation,), self.FITS_RE, None)
```

Comment:
It looks like `num_cores` isn't used anywhere else in `ci_builder`. Do you think you should plan for removal and add a deprecation warning here: https://github.com/lsst-dm/ci_builder/blob/d9eca1844471c6a65d094b3923e0eb6ee72cefa6/python/lsst/ci/builder/ciBuilderLib.py#L128 or could it still be needed in the future?

Reply:
We're not exactly versioning this package strictly, being in `lsst-dm` and all, so I'll throw that question over to @natelust.